flow/embedded_views.h

```diff
@@ -119,15 +119,18 @@ class Mutator {
  private:
   MutatorType type_;
 
+  // TODO(cyanglaz): Remove union.
+  //  https://github.com/flutter/flutter/issues/108470
```

Comment:
Is the SkPath object guaranteed to survive the lifecycle of the mutator? It might be safer to copy the SkPath using a copy constructor since the SkPath copy constructor is documented to be very fast and copy-safe.

Reply:
https://api.skia.org/classSkPath.html#aa30a9363d49c092958d48e30608df367

The SkPath object is 16 bytes I think so the object takes up similar room to SkRect and much less than SkRRect or SkMatrix.

Reply:
> Is the SkPath object guaranteed to survive the lifecycle of the mutator? It might be safer to copy the SkPath using a copy constructor since the SkPath copy constructor is documented to be very fast and copy-safe.

Yeah storing a copy of SkPath is fine with me, but I think shared_ptr for all the them is probably more consistent and fast/light weight too.

Reply:
SkPath is kind of its own shared_ptr for all of the data that matters...